index.js

```diff
@@ -129,6 +129,9 @@ class DockerExecutor extends Executor {
         const piecesParts = imageParser(config.container);
         let buildTag = piecesParts.tag;
         let buildImage = piecesParts.name;
+        const buildTimeout = hoek.reach(config, 'annotations>screwdriver.cd/timeout',
```

Comment:
you can use this `function` to grab the value from annotations https://github.com/screwdriver-cd/executor-base/blob/master/index.js#L198

Reply:
doesn't seem generic, the function parses all annotations.

Reply:
> you can use this `function` to grab the value from annotations https://github.com/screwdriver-cd/executor-base/blob/master/index.js#L198

doesn't seem generic, the function parses all annotations. Will probably need to write another helper function which can take a key